packages/help/help.py

```diff
@@ -0,0 +1,41 @@
+from halibot import HalModule
+
+class Help(HalModule):
+
+	def init(self):
+		self.commands = {
+			"help": self.help_
+		}
+
+	def receive(self, msg):
+		if not msg.body:
+			return
+		body = msg.body.split(" ")
+		func = self.commands.get(body[0][1:])
+		if func:
+			func(body[1:], msg=msg)
+
+	def help_(self, ls, msg=None):
+		if not ls:
+			resp = self.help()
+		elif ls[0] not in self._hal.objects.modules.keys():
+			self.reply(msg, body="Topic '{}' not found!".format(ls[0]))
+			return
+		else:
+			resp = self.invoke(ls[0], "help", args=" ".join(ls[1:]))
```

Comment:
This will throw an exception if the `help` method does not exist, I believe. We should try-catch here.

Reply:
Ah shoot good catch. Artifact of an older version that did the `hasattr()` check before this.